pagecarton/core/library/Ayoola/Application.php

```diff
@@ -1292,48 +1287,36 @@ function_exists( 'http_response_code' ) ? http_response_code(404) : null;
 			}
 			$pagePaths['include'] = 'documents/layout/' . $themeName . '/theme' . $pageThemeFileUrl . '/include';
 			$pagePaths['template'] = 'documents/layout/' . $themeName . '/theme' . $pageThemeFileUrl . '/template';
-			
+
 			//	theme copy
 			$PAGE_INCLUDE_FILE = Ayoola_Loader::getFullPath( $pagePaths['include'], array( 'prioritize_my_copy' => true ) );
 			$PAGE_TEMPLATE_FILE = Ayoola_Loader::getFullPath( $pagePaths['template'], array( 'prioritize_my_copy' => true ) );
 		//	var_export( $pagePaths['include'] );
 		//	var_export( $pagePaths['template'] );
-		//	var_export( $themeName );
-		//	var_export( $pageThemeFileUrl );
-		//	if( stripos( $pageThemeFileUrl, 'pwa' ) === false )
-			{
-			//	exit();
-			}
-		//	var_export( $PAGE_INCLUDE_FILE );
-		//	var_export( $PAGE_TEMPLATE_FILE );
-			if( ! $PAGE_INCLUDE_FILE OR ! $PAGE_TEMPLATE_FILE )
+//			var_export( $pageThemeFileUrl );
+//			var_export( $PAGE_INCLUDE_FILE );
+//			var_export( $PAGE_TEMPLATE_FILE );
+			if( ! $PAGE_INCLUDE_FILE AND ! $PAGE_TEMPLATE_FILE )
 			{
-			//	var_export( $themeName );
 				//	not found
 				return false;
 			}
 			return true;
 		};
 		do
 		{
-			if( ! empty( $_REQUEST['pc_page_layout_name'] ) )     
+			if( ! empty( $_REQUEST['pc_page_layout_name'] ) )
 			{
-			//	var_export( $_REQUEST['pc_page_layout_name'] );
 				if( $previewTheme() )
 				{
 					break;
 				}
 			}
-		//	var_export( Application_Settings_Abstract::getSettings( 'Page', 'default_layout' ) );
-		//	if( stripos( $uri, 'pwa' ) === false )
-			{
-			//	var_export( $PAGE_INCLUDE_FILE );
-		//	exit();
-			}
-				
+
+		//	var_export( $pagePaths );
 			if
-			( 
-				! is_file( $PAGE_INCLUDE_FILE ) OR ! is_file( $PAGE_TEMPLATE_FILE )
+			(
+				! is_file( $PAGE_INCLUDE_FILE ) AND ! is_file( $PAGE_TEMPLATE_FILE )

```

Comment:
Both files must be available for the page to work. So I think the logic should be `OR`